105-server-side-apply-implementation-fg-timelines.md

```diff
@@ -0,0 +1,65 @@
+# Server Side Apply - Implementation Split, Feature Gates, and Timelines
+
+This proposal builds on the original [Server Side Apply proposal](052-k8s-server-side-apply.md), which outlines the technical specification and behavior of Kubernetes Server Side Apply (SSA).
+
+Server Side Apply enables declarative updates to Kubernetes resources by tracking ownership of individual fields through the `.metadata.managedFields` property. 
+This allows multiple controllers (such as Strimzi and other operators like Kyverno) to safely modify different parts of the same resource without overwriting each other’s changes.
```

Comment:
Do you have any exact use-case when Kyverno does this? If not, I would leave this out.

Reply:
I took it from the previous proposal, where the community user mentioned that Kyverno applies `policies.kyverno.io/last-applied-patches`. That's why I mentioned that. 
I can mention it in the the sentence

Reply:
Anyway, I think I can remove the sentence in brackets, as it's not needed.